WORKSPACE

```diff
@@ -1,52 +1,79 @@
 workspace(name = "angular_devkit")
 
+# We get Buildifier from here.
+BAZEL_BUILDTOOLS_VERSION = "0.15.0"
+http_archive(
+    name = "com_github_bazelbuild_buildtools",
+    url = "https://github.com/bazelbuild/buildtools/archive/%s.zip" % BAZEL_BUILDTOOLS_VERSION,
+    strip_prefix = "buildtools-%s" % BAZEL_BUILDTOOLS_VERSION,
+    sha256 = "76d1837a86fa6ef5b4a07438f8489f00bfa1b841e5643b618e01232ba884b1fe",
+)
+
+load("@com_github_bazelbuild_buildtools//buildifier:deps.bzl", "buildifier_dependencies")
+buildifier_dependencies()
+
+# The Go toolchain is used for Buildifier.
+# rules_typescript_dependencies() also tries to load it but we add it explicitely so we
+# don't have hidden dependencies. 
+# This also means we need to load it before rules_typescript_dependencies().
+RULES_GO_VERSION = "0.14.0"
```

Comment:
picking out the versions into variables doesn't seem that useful to me, I've avoided it so we can set up renovate to update automatically (I don't think it understands this)

many repos use `git_repository`, I avoided that because I thought it might fetch the whole git history and therefore wasteful, but I don't know that for sure. It has the advantage of the version appearing once instead of twice.

Reply:
I was using `git_repository` before but asked @gregmagolan and he mentioned that it had been historically flaky (@gregmagolan correct me if I'm wrong).

I think it makes sense to ditch the variable if Renovate doesn't play ball with it though.